.eslintrc.js

```diff
@@ -23,4 +23,9 @@ module.exports = {
     // Place to specify ESLint rules. Can be used to overwrite rules specified from the extended configs
     // e.g. "@typescript-eslint/explicit-function-return-type": "off",
   },
+	overrides: [
+		{
+			files: ['*.ts']
+		}
+	]
```

Comment:
This was committed by accident, but it’s an enhancement nonetheless. This makes ESLint validate `.ts` files without the need to add a command line flag.